molgenis-core-ui/src/main/java/org/molgenis/ui/MolgenisWebAppInitializer.java

```diff
@@ -20,6 +20,9 @@
 
 public class MolgenisWebAppInitializer
 {
+	private static final int MB = 1024 * 1024;
```

Comment:
No, it cannot be a long. The fileSizeThreshold parameter of the MultipartConfigElement constructor is an int cause it shouldn't be higher than maxint.